sql/src/main/java/org/apache/druid/sql/calcite/planner/StatementAttributes.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.calcite.planner;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.calcite.sql.SqlNode;
+
+import javax.annotation.Nullable;
+
+/**
+ * StatementAttributes holds the attributes of a SQL statement. It's used in EXPLAIN PLAN result.
+ */
+public final class StatementAttributes
+{
+  private final String statementKind;
+
+  @Nullable
+  private final SqlNode targetDataSource;
+
+  public StatementAttributes(
+      @JsonProperty("statementKind") final String statementKind,
```

Comment:
Nit: Would you consider `statementType` instead? I find "kind" to be not as easy to understand as "type".

Reply:
ok, done.